npm/publish-docs.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+// ----------------------------------------------------------------------------------------------------------------------
+// This script is intended be run from CI servers, and publish documentation to gh-pages branch
+// ----------------------------------------------------------------------------------------------------------------------
+
+require('shelljs/global');
+
+// Stop on first error
+set('-e');
+
+// run our compile script, discussed above
+exec('npm run build-docs');
```

Comment:
Not really a problem, but I think we `require` modules to run them when using ShellJS. Is this how we've done it in runtime?